packages/core/test/acceptance/extension-point.ts

```diff
@@ -0,0 +1,244 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {Context, BindingScope, inject, Constructor} from '@loopback/context';
+import {Application, ExtensionPoint, ExtensionPointConfig} from '../..';
+
+// tslint:disable:no-any
+interface AuthenticationStrategy {
+  authenticate(credentials: any): Promise<boolean>;
+  config: any;
+}
+
+class LocalStrategy implements AuthenticationStrategy {
+  constructor(@inject('config') public config: any) {}
+
+  authenticate(credentials: any) {
+    return Promise.resolve(true);
+  }
+}
+
+class LDAPStrategy implements AuthenticationStrategy {
+  constructor(@inject('config') public config: any) {}
+
+  authenticate(credentials: any) {
+    return Promise.resolve(true);
+  }
+}
+
+class OAuth2Strategy implements AuthenticationStrategy {
+  constructor(@inject('config') public config: any) {}
+
+  authenticate(credentials: any) {
+    return Promise.resolve(true);
+  }
+}
+
+const AUTH_EXTENSION_POINT = 'authentication.strategies';
+
+class AuthenticationManager extends ExtensionPoint<AuthenticationStrategy> {
```

Comment:
re-posting https://github.com/strongloop/loopback-next/pull/657#discussion_r145364249

Using the name`Manager` is considered as antipattern, see e.g [here](https://stackoverflow.com/questions/1866794/naming-classes-how-to-avoid-calling-everything-a-whatevermanager) and [here](https://stackoverflow.com/questions/5569666/what-is-a-better-name-than-manager-processor-etc).

Let's use a more descriptive name please, for example `AuthenticationHandler`.

On a side note, I find it pretty annoying when my review comments are ignored without any explanation 😞